docs/advanced/transform-content.md

```diff
@@ -0,0 +1,40 @@
+# Transform Content
+
+As we previously stated, you will most likely use MDX through a loader or framework, but it is also possible to interact with or manipulate raw MDX content.  Since MDX is a part of the [unified][unified] ecosystem, there are many utilities we can use to work with MDX.
+
+We can utilize [to-vfile][to-vfile] to read and write MDX files and we can leverage [remark][remark] and [remark-mdx][remark-mdx] to parse and process MDX content.
+
+The following example demonstrates how to put all of that together.
+
+```js
+const {read, write} = require('to-vfile')
+const remark = require('remark');
+const mdx = require('remark-mdx');
+
+(async () => {
+    const path = './path/to/some/file.mdx';
+    const file = await read();
+    const contents = await remark()
+        .use(mdx)
+        // use more plugins or write your own here
```

Comment:
Should we show an example of plugin usage and then display input/output to make it clear what's going on?